bin/rucio

```diff
@@ -247,13 +247,15 @@ def get_client(args):
         creds = None
 
     try:
+        auth_type = args.auth_strategy.lower() if args.auth_strategy else args.auth_strategy
```

Comment:
shouldn't this always just be args.auth_strategy.lower() ?

will save you one call to .lower() later?